lib/elixir/test/elixir/option_parser_test.exs

```diff
@@ -400,3 +400,33 @@ defmodule OptionParserTest do
     assert original == OptionParser.to_argv(opts, switches: [counter: :count])
   end
 end
+
+defmodule OptionsParserDeprecationsTest do
```

Comment:
There are a bunch of warnings now in the OptionParser suite, we need to investigate those too.

Reply:
I'll check them and use either `switch` or `strict` as necessary

Reply:
One suggestion is to move them all to this module and wrap them with capture_io(:stderr).

Reply:
We already have them in other places in the suite, I think we should rather move them here. :)